health/micro-ui/web/packages/modules/campaign-manager/src/components/GenerateXlsx.js

```
@@ -60,7 +61,7 @@ const GenerateXlsx = ({ inputRef, jsonData = [], localeData = [], skipHeader = f
   return (
     <div style={{ display: "none" }}>
       <button ref={inputRef} onClick={handleExport}>
-        {t("WBH_DOWNLOAD_XLS")} {/* ✅ i18n key for “Export XLSX Files” */}
+        {t(LOCALIZATION.WBH_DOWNLOAD_XLS)} {/* ✅ i18n key for “Export XLSX Files” */}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)